a1/src/main/java/ro/utcn/sd/it/a1/service/QuestionManagementSystem.java

```diff
@@ -0,0 +1,86 @@
+package ro.utcn.sd.it.a1.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+import ro.utcn.sd.it.a1.exception.QuestionNotFoundException;
+import ro.utcn.sd.it.a1.model.Question;
+import ro.utcn.sd.it.a1.model.Tag;
+import ro.utcn.sd.it.a1.model.User;
+import ro.utcn.sd.it.a1.persistance.api.QuestionRepository;
+import ro.utcn.sd.it.a1.persistance.api.RepositoryFactory;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+
+@Component
+@Service
+@RequiredArgsConstructor
+public class QuestionManagementSystem {
+    private final RepositoryFactory repositoryFactory;
+
+
+    //QUESTION OPERATIONS
```

Comment:
The fact that you have sections in this class should be a hint that you should split the class into several smaller classes.